docs/reference/sdk-guidelines.mdx

```diff
@@ -0,0 +1,679 @@
+<br />
+<div align="center">
+  <a href="">
+    <img src="./" alt="Logo" width="300" height="auto">
+  </a>
+  <p align="center"> <br />
+    <a href=""><strong>View on GitHub »</strong></a> <br /><br />
+    <a href="https://docs.onflow.org/sdk-guidelines/">SDK Specifications</a> ·
+    <a href="">Contribute</a> ·
+    <a href="">Report a Bug</a>
+  </p>
+</div><br />
+
+## Overview 
+
+This reference documents all the methods available in the SDK, and explains in detail how these methods work.
+SDKs are open source, and you can use them according to the licence.
+
+The library client specifications can be found here:
+
+// TODO specs here
```

Comment:
will spec generation be possible? jsdoc?

Reply:
Fixed this to link to reference, but jsdoc will be coming. Just not ready yet.